README.md

```diff
@@ -1712,6 +1712,9 @@ IEEE Sensors Journal, 20(11):6170-6181, 2020.
 - <a name="Chen21tfis"></a>Chen, G., Liu, P., Liu, Z., Tang, H., Hong, L., Dong, J., Conradt, J., Knoll, A.,  
 *[NeuroAED: Towards Efficient Abnormal Event Detection in Visual Surveillance With Neuromorphic Vision Sensor](https://doi.org/10.1109/TIFS.2020.3023791)*,  
 IEEE Trans. Information Forensics and Security (TIFS), 16:923-936, 2021.
+- <a name="Liu_21_IJCAI"></a>Liu, Q., Xing, D., Tang, H., Ma, D., Pan, G.,  
```

Comment:
key: Liu21ijcai (if it has not been used before)

Reply:
These should be resolved, feel free to review the changes again.